src/openrct2/rct12/RCT12.cpp

```diff
@@ -483,7 +483,7 @@ bool is_user_string_id(rct_string_id stringId)
 
 std::string RCT12::RemoveFormatCodes(const std::string_view& s)
 {
-    constexpr auto RCT12_MULTIBYTE_PREFIX = (char)(uint8_t)0xFF;
+    constexpr auto RCT12_MULTIBYTE_PREFIX = static_cast<char>(static_cast<uint8_t>(0xFF));
```

Comment:
```suggestion
    constexpr char RCT12_MULTIBYTE_PREFIX = 0xFF;
```
?

Reply:
I'm doing this update in batches, so I don't pay too much attention on what could actually be simpler, because it's easier to go after named casts later, than C-style casts.

tl;dr Doing your change yields a warning that becomes an error

Reply:
Because 0xFF is not of type char, '\xFF' would work.